lib/internal/fs/utils.js

```diff
@@ -749,6 +750,7 @@ const validateCpOptions = hideStackFrames((options) => {
   validateBoolean(options.force, 'options.force');
   validateBoolean(options.preserveTimestamps, 'options.preserveTimestamps');
   validateBoolean(options.recursive, 'options.recursive');
+  validateBoolean(options.verbatimSymlinks, 'options.verbatimSymlinks');
```

Comment:
I believe that this function should throw if `options.dereference && options.verbatimSymlinks`. (Unless that combination somehow makes sense?)

Reply:
You are right, that combination does not make sense. I have applied the changes, making it throw an `ERR_INVALID_ARG_VALUE` error if those values were enabled together, let me know if my proposed approach makes sense to you or if you would have preferred it to be done another way.

In addition to that, I've also implemented a test for checking this specific case.